### PR TITLE
[RELEASE] Prepare version 1.0.0

### PR DESCRIPTION
**Pull request by PhilippKuhlmay**

### CODE REVIEW

File: ext_emconf.php [hunk 1]
```diff
@@ -26,12 +26,12 @@
             'crawler' => '9.0.0-9.9.99'
         ],
     ],
-    'state' => 'alpha',
+    'state' => 'beta',
```
Comment (brotkrueml) [inline ext_emconf.php:29]: In would say: stable. It is a version 1.0.0.

Comment (PhilippKuhlmay) [reply ext_emconf.php]: Changed to stable

File: ext_emconf.php [hunk 2]
```diff
@@ -26,12 +26,12 @@
             'crawler' => '9.0.0-9.9.99'
         ],
     ],
-    'state' => 'alpha',
+    'state' => 'stable',
     'clearCacheOnLoad' => 1,
     'author' => 'Philipp Kuhlmay',
```
Comment (brotkrueml) [inline ext_emconf.php:31]: One more: I think you should add Tomas and me also to the authors ;-)

Comment (PhilippKuhlmay) [reply ext_emconf.php]: I added you and Tomas

**Review by brotkrueml — APPROVED** [review]

**Comment by tomasnorre** [comment]
Feel free to merge.

**Review by tomasnorre — APPROVED** [review]

**Pull request merged**